### PR TITLE
Fixing small null ref errors that i found from the changes in Pr #1420

### PR DESCRIPTION
**Pull request by akshar2401**
Forgot to handle few cases where null ref could be thrown in this pr https://github.com/microsoft/Power-Fx/pull/1420. This pr handles those null ref scenarios. It is a small pr.

### CODE REVIEW

File: src/libraries/Microsoft.PowerFx.LanguageServerProtocol/LanguageServer/LanguageServer.cs [hunk 1]
```diff
@@ -493,7 +493,12 @@ private void HandleSemanticTokens<T>(string id, T semanticTokensParams, string m
 
             var tokenTypesToSkip = ParseTokenTypesToSkipParam(queryParams?.Get("tokenTypesToSkip"));
             var scope = _scopeFactory.GetOrCreateInstance(semanticTokensParams.TextDocument.Uri);
-            var result = scope.Check(expression);
+            var result = scope?.Check(expression);
+            if (scope == null)
+            {
+                SendEmptySemanticTokensResponse(id);
+                return;
+            }
```
Comment (CarlosFigueiraMSFT) [inline src/libraries/Microsoft.PowerFx.LanguageServerProtocol/LanguageServer/LanguageServer.cs:502]: You can move this check to before the previous line, and after that you won't need to use the `?` null-propagating operator in the Check call.

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.LanguageServerProtocol/LanguageServer/LanguageServer.cs (hunk 1 above)
Comment (akshar2401) [reply src/libraries/Microsoft.PowerFx.LanguageServerProtocol/LanguageServer/LanguageServer.cs:502]: This is a nice catch. I meant to check for result == null and not just scope.

**Review by jas-valgotar — DISMISSED** [review]

**Review by jas-valgotar — APPROVED** [review]

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**